python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -162,10 +203,11 @@ def read_fn():
         self.last_update_time = 0.0
         self.update_interval_s = update_interval_s
 
-        # Tracks active worker nodes
-        self.workers = []
+        # Keeps track of pending and running nodes
+        self.nodes: Optional[NonTerminatedNodes] = None
```

Comment:
slight preference for naming this param `self.non_terminated_nodes`

Reply:
done